### PR TITLE
add Man page for launch.sh

### PR DESCRIPTION
**Pull request by JanZerebecki**

### CODE REVIEW

File: docs/novnc_server.1 [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.TH novnc_server 1  "July 14, 2020" "version 1.2.0" "USER COMMANDS"
+
+.SH NAME
+novnc_server - noVNC proxy server
```
Comment (samhed) [inline docs/novnc_server.1:4]: I'd consider renaming it to novnc_proxy, otherwise, people might think this will launch a VNC server.

Comment (JanZerebecki) [reply docs/novnc_server.1]: Done.

**Review by CendioOssman — CHANGES_REQUESTED** [review]
Looks good, thanks. And sorry for the delay.

There are two references to the old launch around though. Could you please fix them?

And please squash everything together to a single commit and do a force push update.

**Comment by CendioOssman** [comment]
Ping @JanZerebecki .

**Comment by JanZerebecki** [comment]
Requested changes done and rebased.

**Review by samhed — APPROVED** [review]
Looks good and works well :+1:

**Pull request merged**